### ISSUE TITLE
FastbootAdapter requires maxAge to be set

### ISSUE THREAD

**Issue by billdami** [embermap/ember-data-storefront#383]
Not sure if this was intentional, but it looks like the shoebox data expiration feature that was added via #369 that was added in the latest release causes the adapter to never use cached request data if a `storefront.maxAge` environment config value is not set. The current documentation makes it sound like this configuration is optional, which, imo, it probably should be. E.g.

```js
isDateValid(createdString, maxAgeMinutes) {
    return !maxAgeMinutes || (new Date() - new Date(createdString)) / 1000 / 60 < maxAgeMinutes;
}
```

**Comment by st-h** [comment]
@billdami thanks. I thought I checked that it keeps the previous behaviour - but looking at the code now, I think you are right.  I think your fix is reasonable. Could you please submit a PR?

**Comment by xg-wang** [comment]
I think we can make the first request goes through shoebox, the followings respect `maxAge`

**Comment by ryanto** [comment]
Released as `0.18.1` thanks to @xg-wang
